mrtt-ui/src/components/SiteInterventions.js

```diff
@@ -265,6 +369,143 @@ function SiteInterventionsForm() {
           <ErrorText>{errors.biophysicalInterventionsUsed?.message}</ErrorText>
         </FormQuestionDiv>
 
+        {displayMangroveSpeciesUsed() ? (
```

Comment:
see, here is it unclear that this is a boolean. It reads like it might render something (although would be weird as the condition of a ternary, but still)

Reply:
updated to `isMa`ngroveSpeciesUsedShowing